### ISSUE TITLE
cannot import name 'regionprops_table' from 'skimage.measure' 

### ISSUE THREAD

**Issue by tinawiegand** [scikit-image/scikit-image#4722]
## Description
Hei I have trouble using the function regionprops_table: Error upon importing "Import Error" or upon usage "AttributeError: module 'skimage.measure' has no attribute 'regionprops_table'"
As suggested in the issue 432 I deinstalled and reinstalled using conda - still the same issue. Sorry I do not know how to prepare fresh environment /what a fresh venv is o.O
Cheers
Tina

## Way to reproduce
```python
import numpy as np
import skimage.io as io
import skimage.measure
# uncommanded this produces import error:
# from skimage.measure import label, regionprops_table

image = np.zeros((512,512))
image_label=skimage.measure.label(image)
# this produces attribute error:
regions = skimage.measure.regionprops_table(image_label) 
```

## Version information
```python 3.7
# Paste the output of the following python commands
from __future__ import print_function
import sys; print(sys.version)
import platform; print(platform.platform())
import skimage; print("scikit-image version: {}".format(skimage.__version__))
import numpy; print("numpy version: {}".format(numpy.__version__))
```

```python
# your output here
3.7.2 (default, Dec 29 2018, 00:00:04) 
[Clang 4.0.1 (tags/RELEASE_401/final)]
Darwin-18.5.0-x86_64-i386-64bit
scikit-image version: 0.15.0
numpy version: 1.18.4
```

**Comment by emmanuelle** [comment]
Hi Tina! `regionprops_table` has been introduced in skimage 0.16 so you need to upgrade your version, here you have version 0.15.

**Comment by emmanuelle** [comment]
Did you run `conda install -c conda-forge scikit-image` ? You need to get scikit-image from `conda-forge`.

**Comment by tinawiegand** [comment]
Ah yes I did - wired it did not take the latest release!? Let me try again -thank you already!

**Comment by tinawiegand** [comment]
did it again with your command - still 0.15 :( 
I tried the development installation but `pip install -e ` inside the directory scikit-image gives following Error:

> 
>   ERROR: Command errored out with exit status 1:
>      command: /anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/Users/wiegand/scikit-image/setup.py'"'"'; __file__='"'"'/Users/wiegand/scikit-image/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info
>          cwd: /Users/wiegand/scikit-image/
>     Complete output (7 lines):
>     Traceback (most recent call last):
>       File "<string>", line 1, in <module>
>       File "/Users/wiegand/scikit-image/setup.py", line 234, in <module>
>         'build_ext': openmp_build_ext(),
>       File "/Users/wiegand/scikit-image/setup.py", line 58, in openmp_build_ext
>         from numpy.distutils.command.build_ext import build_ext
>     ModuleNotFoundError: No module named 'numpy'
>     ----------------------------------------
> ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
>

**Comment by tinawiegand** [comment]
Alright - reinstallation with the normal `pip install scikit-image` command did the job :)
Thanks again!
